content/c-sharp/concepts/math-functions/terms/sin/sin.md

```diff
@@ -22,15 +22,17 @@ The **`Math.Sin()`** class method returns the sine of a given angle.
 Math.Sin(angle);
 ```
 
-The `Math.Sin()` method takes only one `double` type parameter - `angle` - an angle in radians. The `Math.Sin()` method returns the sine of the `angle` as a `double` value  or `NaN` (not a number) if the `angle` equals to:
+The `Math.Sin()` method takes only one parameter, `angle`, an angle in radians of type `double`. The method returns the sine of the `angle` as a `double` value  or `NaN` (not a number) if the value of `angle` equals:
 
-- `NaN`,
-- `NegativeInfinity`, or
-- `PositiveInfinity`.
+- `NaN`
+- `NegativeInfinity`
+- `PositiveInfinity`
+
+> **Note:** Depending on the used operating system or architecture, the exact result or the input range of the `Math.Sin()` method may differ due to the differences in the C runtime environment.
```

Comment:
As I mentioned in the other entry I think we can move this note down and edit out the rounding step from the examples. Thanks